utils/rrdtool1/Makefile

```diff
@@ -17,6 +17,9 @@ PKG_SOURCE_URL:=http://oss.oetiker.ch/rrdtool/pub/archive/rrdtool-1.0.x
 PKG_HASH:=42aa7c213dedbd95d33ca84d92f4187880f7e96062c6a3fb05bfb16f77ba2a91
 PKG_MAINTAINER:=Jo-Philipp Wich <jo@mein.io>
 
+PKG_LICENSE:=GPL-2.0
```

Comment:
This is outdated SPDX License Identifier. Should be only or later
See: https://spdx.org/licenses/